test/notaryUtils.test.js

```diff
@@ -50,6 +50,50 @@ describe('notaryUtils.setupCase', function () {
     assert.equal(caseID, notary + '/cases/' + pathParts[pathParts.length - 1])
     caseNock.done()
   })
+
+  it('creates a case ID and then returns it on 200', function * () {
```

Comment:
In the above test case it tests automatically creating the ID and here we're testing passing in an id right?